projects/go-lib/src/styles/_globals.scss

```diff
@@ -24,3 +24,13 @@
     width: 0;
   }
 }
+
+.go-hr {
+  border: 1px solid $theme-light-app-bg;
```

Comment:
So we'll actually need to change this around a bit. Because this is border, this element actually ends up being 2px tall because of the top/bottom borders. The way around this is to hide the border and apply a height / background.

```
background: #f0f0f0;
border: 0;
height: 1px;
```